features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -489,3 +489,8 @@ CREATE TABLE IF NOT EXISTS FIDO_DEVICE_STORE (
         KEY_HANDLE VARCHAR(200) NOT NULL,
         DEVICE_DATA LONGVARCHAR NOT NULL,
       	PRIMARY KEY (DOMAIN_ID, TENANT_ID, USER_NAME, KEY_HANDLE));
+
+CREATE TABLE IF NOT EXISTS IDN_TENANT_CONFIG (
```

Comment:
No need to have TENANT part in table name. Shall we rename it to IDN_RESIDENT_IDP. Expectation is that it will contain all the properties related to RESIDENT IDP in the future. We haven't got a table like this yet.